### PR TITLE
Create harbour-matkakortti-zh_CN.ts

### PR DESCRIPTION
**Pull request by dashinfantry**
New

### CODE REVIEW

**Comment by dashinfantry** [comment]
Can you check this new Chinese translation

**Comment by dashinfantry** [comment]
I change that 2 strings about "load".It seems I misunderstand them:)

**Comment by dashinfantry** [comment]
The Chinese(汉语) nouns don't have singular  and plural:)

File: translations/harbour-matkakortti-zh_CN.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<TS version="2.1" language="zh_CN">
+<!DOCTYPE TS>
+<TS version="2.1">
```
Comment (monich) [inline translations/harbour-matkakortti-zh_CN.ts:3]: First 3 lines have to look as follows:
```xml
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="zh_CN">
```

**Pull request closed without merging**